storage/cloud-client/README_TEST.md

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Why `README_TEST`? If users need this to run the tests, we should mention it in the main README

Reply:
Sure, I can do it